core-services/openshift-priv/README.md

```diff
@@ -0,0 +1,3 @@
+# OpenShift Private
+
+Manifests and configuration of the openshift-priv organization.
```

Comment:
This description is not useful. The point of `core-services` directory is to hold manifests and configuration. The point of `openshift-priv` subdirectory is to hold that for `openshift-priv`. Let's do better with our docs.